x-pack/plugins/apm/server/feature.ts

```diff
@@ -6,13 +6,15 @@
 
 import { i18n } from '@kbn/i18n';
 import { AlertType } from '../common/alert_types';
+import { DEFAULT_APP_CATEGORIES } from '../../../../src/core/server';
 
 export const APM_FEATURE = {
   id: 'apm',
   name: i18n.translate('xpack.apm.featureRegistry.apmFeatureName', {
     defaultMessage: 'APM',
```

Comment:
@jportner Like this? 

```suggestion
    defaultMessage: 'APM and Client Side Monitoring',
```

Lgtm. @shahzad31 ?

Reply:
Seems fine

Reply:
Yes exactly!

Reply:
I like it - thanks!